### DISCUSSION TITLE
JuiceFS挂载的目录通过NFS-Ganesha export出去之后写入速度为什么非常慢

### DISCUSSION THREAD

**Discussion by panbian** [juicedata/juicefs#3710] [Q&A]
我将JuiceFS挂载到本地的某个目录a上，然后通过NFS-Ganesha export出去。如果juicefs mount不开启writeback模式，那么NFS客户端大文件上传就会很慢，开启writeback模式后，NFS客户端大文件上传就很快，速度差距5~6倍，请问是什么原因呢？谢谢！

**Comment by zhijian-pro** [comment]
直接往 juicefs 挂载点下的 a 目录写速度怎么样？
